bunq/sdk/json/anchored_object_model_adapter.py

```diff
@@ -0,0 +1,70 @@
+from bunq import AnchoredObjectInterface
+from bunq.sdk.exception.bunq_exception import BunqException
+from bunq.sdk.json import converter
+from bunq.sdk.model.generated import endpoint
+from bunq.sdk.model.generated import object_
+
+
+class AnchoredObjectModelAdapter(converter.JsonAdapter):
+    _ERROR_MODEL_NOT_FOUND = '{} is not in endpoint nor object.'
+
+    __STRING_FORMAT_UNDERSCORE = '_'
+
+    _override_field_map = {
+        'ScheduledPayment': 'SchedulePayment',
+        'ScheduledInstance': 'ScheduleInstance',
+    }
+
+    @classmethod
+    def deserialize(cls, cls_target, obj_raw):
+        """
+        :type cls_target: BunqModel
```

Comment:
Shouldn't this be `Type[BunqModel]` ?

Reply:
Do you mean should to doc be `:type cls_target: Type[BunqModel]`?

Reply:
afaict yes, the object you're accepting here is the class itself right? not an instance of that class.
According tot pep484 you can express the class of `Foo` as `Type[Foo]`